src/SprykerEco/Yves/FactFinderWebComponents/FactFinderWebComponentsConfig.php

```diff
@@ -0,0 +1,260 @@
+use SprykerEco\Shared\FactFinderWebComponents\FactFinderWebComponentsConstants;
+
+class FactFinderWebComponentsConfig extends AbstractBundleConfig implements WebComponentsConfigInterface
+{
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getCommunicationConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::COMMUNICATION_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getSearchBoxConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::SEARCH_BOX_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getBreadcrumbConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::BREADCRUMB_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getSearchButtonConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::SEARCH_BUTTON_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getRecordListConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::RECORD_LIST_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getCheckoutTrackingConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::CHECKOUT_TRACKING_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getRecordConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::RECORD_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getSuggestConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::SUGGEST_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getAsnGroupWidgetConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::ASN_GROUP_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getAsnGroupElementConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::ASN_GROUP_ELEMENT_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getAsnRemoveAllFilterConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::ASN_REMOVE_ALL_FILTER_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getAsnSliderConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::ASN_SLIDER_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getAsnSliderControlConfig(): array
```

Comment:
Weird name. Please rename to same as constant name.

Reply:
Method removed it duplicates functionality of getRecordComponentConfig() method.